mne_bids/write.py

```diff
@@ -965,21 +967,31 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     _mkdir_p(os.path.dirname(bids_fname))
 
     if verbose:
-        print('Copying data files to %s' % bids_fname)
-
-    if ext not in ALLOWED_EXTENSIONS:
-        raise ValueError('ext must be in %s, got %s'
-                         % (''.join(ALLOWED_EXTENSIONS), ext))
+        print('Copying data files to %s' % op.splitext(bids_fname)[0])
 
+    convert = ext not in ALLOWED_EXTENSIONS[kind]
     # Copy the imaging data files
-    if ext in ['.fif']:
+    if convert and (kind == 'eeg' or kind == 'ieeg'):
+        if verbose:
+            warn('Converting data files to BrainVision format')
+        requires_version('pybv', '0.2.0')
```

Comment:
I think you should use `check_version` here. See how we do it in MNE: https://github.com/mne-tools/mne-python/blob/9e588fca0007e8bda43c8a1349d679aaa71c0768/mne/preprocessing/ica.py#L330